__tests__/app.test.js

```diff
@@ -74,6 +74,7 @@ describe("GET /api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: 11,
         });
```

Comment:
When following Test Driven Development principles, it's not good practice to go back and change old tests. Preferably you would write an entire new test to check that comment_count is added to your responses